glustercli/cmd/cluster-options.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	helpClusterOptionCmd    = "Gluster Cluster Option Management"
+	helpClusterOptionSetCmd = "Set Cluster Option"
+	helpClusterOptionGetCmd = "Get Cluster Option"
+)
+
+func init() {
+	clusterCmd.AddCommand(clusterOptionGetCmd)
+	clusterCmd.AddCommand(clusterOptionSetCmd)
+}
+
+var clusterCmd = &cobra.Command{
+	Use:   "cluster",
+	Short: helpClusterOptionCmd,
+}
+
+var clusterOptionSetCmd = &cobra.Command{
+	Use:   "set <option> <value> [<option> <value>]...",
+	Short: helpClusterOptionSetCmd,
+	Args:  clusterSetCmdArgs,
+	Run:   clusterSetCmdRun,
+}
+
+func clusterSetCmdArgs(cmd *cobra.Command, args []string) error {
+	// Ensure we have enough arguments for the command
+	if len(args) < 2 {
+		return errors.New("need at least 2 arguments")
+	}
+
+	// Ensure we have a proper option-value pairs
+	if (len(args) % 2) != 0 {
+		return errors.New("needs '<option> <value>' to be in pairs")
+	}
+
+	return nil
+}
+
+func clusterSetCmdRun(cmd *cobra.Command, args []string) {
+	options := args[:]
+	if err := clusterOptionJSONHandler(cmd, options); err != nil {
+		if GlobalFlag.Verbose {
+			log.WithError(err).Error("cluster option set failed")
+		}
+		failure("Cluster option set failed", err, 1)
+	} else {
+		fmt.Printf("Options set successfully \n")
```

Comment:
why can't you use `fmt.Println()`?

Reply:
Would it make a big difference?

Reply:
we already have a function that provides the same functionality, what's the issue in using default function?